### ISSUE TITLE
How to pretrain baseline classifier?

### ISSUE THREAD

**Issue by TwoBranchDracaena** [AntreasAntoniou/MatchingNetworks#1]
Hi, Antreas,

Thank you for the code!

I have a question about the experiment setting in chapter 4.1:
'The baseline classifier was trained to classify an image into one of the original classes present in the training data set'

For my understanding, this means to first pre-train the baseline classifier on all the training classes in general manner, that's a 1200 classes classification for Omniglot. Then, use the feature in this baseline classifier(before softmax) to train in one-shot manner.

What do you think of it?

Regards,

Xiaolu

**Comment by AntreasAntoniou** [comment]
I actually tried that. But for the one-shot learner you don't really need to do that. You can train the one shot network directly without the pretrained features. In fact attempting to learn standard classifiers on omniglot results to terrible results.
